src/newsArticle.js

```diff
@@ -0,0 +1,17 @@
+class newsArticle {
```

Comment:
i've been told it's good practice to have seperate functions for when you want to return something from the constructor - like a showWebUrl() that just returns this._webUrl